src/components/select/select.spec.js

```diff
@@ -1486,6 +1506,29 @@ describe('<md-select>', function() {
     });
   });
 
+  // Only test with custom components on AngularJS 1.5+
+  if (angular.version.major === 1 && angular.version.minor >= 5) {
+    describe('with custom components', function() {
+
+      if (angular.version.major === 1 && angular.version.minor >= 7) {
```

Comment:
The `angular.version.major` check is redundant (since it has been checked above).
At the moment, there is only one test, so the two `if` blocks could be merged.